db/models.py

```diff
@@ -50,3 +56,74 @@
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey("User",
+                             on_delete=models.CASCADE,
+                             related_name="orders")
+
+    class Meta:
+        ordering = ["-created_at"]
```

Comment:
Code Style: Implement `class Meta` after you defined all attributes. In this case, it's better to place it after the `user` attribute.